algorithms/kernel/svm/svm_train_boser_impl.i

```diff
@@ -514,17 +166,19 @@ void SVMTrainTask<algorithmFPType, ParameterType, cpu>::WSSjLocal(const size_t j
  *              M(alpha) = min(-y[i]*grad[i]): i belongs to I_low(alpha)
  */
 template <typename algorithmFPType, typename ParameterType, CpuType cpu>
-Status SVMTrainTask<algorithmFPType, ParameterType, cpu>::WSSj(size_t nActiveVectors, algorithmFPType tau, int Bi, algorithmFPType GMax, int & Bj,
-                                                               algorithmFPType & delta, algorithmFPType & res) const
+services::Status SVMTrainTask<algorithmFPType, ParameterType, cpu>::WSSj(size_t nActiveVectors, algorithmFPType tau, int Bi, algorithmFPType GMin,
+                                                                         int & Bj, algorithmFPType & delta, algorithmFPType & res) const
 {
+    DAAL_ITTNOTIFY_SCOPED_TASK(findMaximumViolatingPair.WSSj);
+
     Bj                    = -1;
     algorithmFPType fpMax = MaxVal<algorithmFPType>::get();
-    algorithmFPType GMin  = fpMax; // some big positive number
-    algorithmFPType GMin2 = fpMax;
+    algorithmFPType GMax  = -fpMax; // some big positive number
```

Comment:
comment is not aligned with sign of variable.

Reply:
Fixed